app/pages/home-not-logged-in/research.jsx

```diff
@@ -43,7 +39,7 @@ counterpart.registerTranslations('en', {
 const HomePageResearch = (({ count, screenWidth, showDialog, volunteerCount }) =>
   <section className="home-research">
     <Translate className="tertiary-kicker" component="h2" content="researchHomePage.works" />
-    <span className="class-counter">{count.toLocaleString()}</span>
+    <span className="class-counter">{count.toString().replace(/.(?=(?:.{3})+$)/g, '$&,')}</span>
```

Comment:
Our policy generally is we support current version and the most recent previous version only. MDN reports that Safari 10 should support it. If the issue is being seen in Safari 9, we could look into polyfilling it. Looks like the service we use will do it, if we specify: https://github.com/Financial-Times/polyfill-service/issues/359